src/pages/index.js

```diff
@@ -0,0 +1,328 @@
+editClose.src = closeLogo;
+newClose.src = closeLogo;
+avatarImage.src = avatarImg;
+editAvatar.src = whitePencil;
+avatarClose.src = closeLogo;
+deleteClose.src = whiteCloseLogo;
+// const initialCards = [
+//   {
+//     name: "Golden Gate Bridge",
+//     link: "   https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/7-photo-by-griffin-wooldridge-from-pexels.jpg",
+//   },
+//   {
+//     name: "Val Thorens",
+//     link: " https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/1-photo-by-moritz-feldmann-from-pexels.jpg",
+//   },
+//   {
+//     name: "Restaurant terrace",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/2-photo-by-ceiline-from-pexels.jpg",
+//   },
+//   {
+//     name: "An outdoor cafe",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/3-photo-by-tubanur-dogan-from-pexels.jpg",
+//   },
+//   {
+//     name: "A very long bridge, over the forest and through the trees",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/4-photo-by-maurice-laschet-from-pexels.jpg",
+//   },
+//   {
+//     name: "Tunnel with morning light",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/5-photo-by-van-anh-nguyen-from-pexels.jpg",
+//   },
+//   {
+//     name: "Mountain house",
+//     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/6-photo-by-moritz-feldmann-from-pexels.jpg",
+//   },
+// ];
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "2a33eb51-29e1-461c-a628-84dfa7ed62f0",
+    "Content-Type": "application/json",
+  },
+});
+//destructure the second item in the callback of the .then()
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardList.append(cardElement);
+    });
+    profileNameElement.textContent = userInfo.name;
+    profileJobElement.textContent = userInfo.about;
+    avatarElement.src = userInfo.avatar;
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+//modal elements
+const modals = document.querySelectorAll(".modal");
+const avatarElement = document.querySelector(".profile__avatar");
+const profileNameElement = document.querySelector(".profile__name");
+const profileJobElement = document.querySelector(".profile__description");
+const profileEditButton = document.querySelector(".profile__button-secondary");
+const profilePostButton = document.querySelector(".profile__button-primary");
+
+//Edit Elements
+const editModal = document.querySelector("#edit-modal");
+const editModalCloseButton = editModal.querySelector(".modal__button");
+const editModalSubmitButton = editModal.querySelector(
+  "#modal__edit-submit-button"
+);
+const editFormElement = editModal.querySelector("#modal__edit-form");
```

Comment:
**COULD BE IMPROVED**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms